stores/sql/sqlite/migrations/main/migration_00014_hosts_resolvedaddresses.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE hosts DROP COLUMN subnets;
+ALTER TABLE hosts ADD resolved_addresses varchar(255);
+
```

Comment:
I don't think this migration works. Before we merge this we need to manually verify it against sqlite and mysql.

Reply:
```
2024-08-14T13:01:33-04:00       INFO    sql     Using SQLite version 3.45.1
2024-08-14T13:01:33-04:00       INFO    sql     performing main migration '00014_hosts_resolvedaddresses'
2024-08-14T13:01:33-04:00       INFO    sql     migration '00014_hosts_resolvedaddresses' complete
```

```
2024-08-14T13:24:43-04:00       INFO    sql     Using MySQL version 11.1.2-MariaDB-debug
2024-08-14T13:24:43-04:00       INFO    sql     performing main migration '00014_hosts_resolvedaddresses'
2024-08-14T13:24:43-04:00       INFO    sql     migration '00014_hosts_resolvedaddresses' complete
```